apollo-router-core/src/query_planner/mod.rs

```diff
@@ -1,6 +1,6 @@
-mod caching_query_planner;
-mod model;
-mod router_bridge_query_planner;
+pub(crate) mod caching_query_planner;
+pub(crate) mod model;
+pub(crate) mod router_bridge_query_planner;
```

Comment:
I think only the model needs to be crate public.

Reply:
Fixed in 33e03a6